src/app.rs

```diff
@@ -655,6 +663,20 @@ impl<'a, 'b, T> ApplicationBuilder<'a, 'b, T> {
         self
     }
 
+    /// Sets the maximum frames per second of this game.
+    ///
+    /// # Parameters
+    ///
+    /// `max_fps`: the maximum frames per second this game will run at.
+    ///
+    /// # Returns
+    ///
+    /// This function returns the ApplicationBuilder after modifying it.
+    pub fn with_max_fps(mut self, max_fps: u32) -> Self {
```

Comment:
One might expect that setting `max_fps = 0` removes the cap. (How) should we address that?

Reply:
Well removing the cap is quite dangerous as we've proven.  I'd rather not build around enabling that.  If someone really hates the cap they could set it to something absurd like 9999.

Reply:
Yes, but atm using `0` will cause the result of the division to be infinite. So my concern is that the convention to use `0` for limits as a way to disable it could cause confusion here. So if we don't implement that, we need documentation for that behavior.

Reply:
Yeah that's a good point.  Especially if this is going in the config file we ought to handle for that.

Reply:
```rust
if max_fps == 0 {
    self.max_fps = std::u32::MAX;
} else {
    self.max_fps = max_fps
}
```

That seems like a real simple way to enable using 0 while not building around having no FPS cap.

Reply:
Fixed.

Reply:
Such special cases could use documentation anyway.